_layouts/blog-index.html

```diff
@@ -4,7 +4,6 @@
 
 <section class="items">
   {% if site.prexian.site_type == "hub" %}
-    {% include assets/symbol.svg %}
```

Comment:
What's the reason for removing this file?

Reply:
Yep, we don't need to remove this, seems like it was removed by mistake.